<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DatasetConverter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditDatasetDir">
            <property name="text">
             <string>F:/Dataset/PascalVOC/dataset/pascalvoc2007/VOCtrainval_06-Nov-2007</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSelectDatasetFolder">
            <property name="text">
             <string>SelectDatasetFolder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditSelectSaveFolder">
            <property name="text">
             <string>F:/Dataset/PascalVOC/dataset/pascalvoc2007</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSelectSaveFolder">
            <property name="text">
             <string>SelectSaveFolder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>OutPrefixName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOutPrefixName">
          <property name="text">
           <string>voc_train</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>AnnotationFolderName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAnnotationFolder">
          <property name="text">
           <string>Annotations</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ImageFolderName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditImageFolder">
          <property name="text">
           <string>JPEGImages</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>SampelsPerFile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSampelsPerFile">
          <property name="text">
           <string>200</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBoxShuffling">
          <property name="text">
           <string>shuffling</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStart">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>LabelText</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditLabel">
        <property name="plainText">
         <string>none, 0, Background
aeroplane, 1, Vehicle
bicycle, 2, Vehicle
bird, 3, Animal
boat, 4, Vehicle
bottle, 5, Indoor
bus, 6, Vehicle
car, 7, Vehicle
cat, 8, Animal
chair, 9, Indoor
cow, 10, Animal
diningtable, 11, Indoor
dog, 12, Animal
horse, 13, Animal
motorbike, 14, Vehicle
person, 15, Person
pottedplant, 16, Indoor
sheep, 17, Animal
sofa, 18, Indoor
train, 19, Vehicle
tvmonitor, 20, Indoor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
